pulsar-common/src/main/java/org/apache/pulsar/common/util/SecurityUtility.java

```diff
@@ -124,7 +124,12 @@ private static Provider loadConscryptProvider() {
             conscryptClazz = Class.forName("org.conscrypt.Conscrypt");
             conscryptClazz.getMethod("checkAvailability").invoke(null);
         } catch (Throwable e) {
-            log.warn("Conscrypt isn't available. Using JDK default security provider.", e);
+            if (e.getCause() instanceof UnsatisfiedLinkError) {
```

Comment:
@codelipenghui @nodece Do we have an exception util like Flink's [`findThrowable`](https://github.com/apache/flink/blob/99c2a415e9eeefafacf70762b6f54070f7911ceb/flink-core/src/main/java/org/apache/flink/util/ExceptionUtils.java#L466-L482)? I'm afraid that the error may be changed to differ wrap level.

Reply:
No.